### PR TITLE
Newsletter-282-Translate-into-French

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

**Comment by Copinmalin** [comment]
@bitschmidty @murchandamus  can you help me with this error please ?

```
## Check that posts declare a slug, see issue #155 and PR #156

! git --no-pager grep -L "^slug: " _posts

_posts/fr/newsletters/2023-12-20-newsletter.md
```

File: _posts/fr/newsletters/2023-12-20-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,799 @@
...
+clés de l'opcode proposé `OP_CHECKTEMPLATEVERIFY`, bien que la version MATT soit moins efficace en termes d'espace. Halseth en a
+également profité pour présenter aux lecteurs un outil qu'il avait développé, [tapsim][], qui permet de déboguer les transactions
+Bitcoin et les [tapscript][topic tapscript].
+
+En [j]uin][jun specsf], Robin Linus a décrit comment les utilisateurs pouvaient verrouiller des fonds aujourd'hui, les utiliser sur
+une sidechain pendant longtemps, puis permettre aux destinataires des fonds sur la sidechain de les retirer efficacement sur Bitcoin
+ultérieurement---mais seulement si les utilisateurs de Bitcoin décident finalement de modifier les règles de consensus d'une
+certaine manière.
+Cela pourrait permettre aux utilisateurs désireux de prendre un risque financier de commencer immédiatement à utiliser leurs fonds avec
+les nouvelles fonctionnalités de consensus qu'ils désirent, tout en offrant un moyen de faire revenir ces fonds plus tard sur le mainnet
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2023-12-20-newsletter.md:182]:
```suggestion
les nouvelles fonctionnalités de consensus qu'ils désirent, tout en offrant un moyen de faire revenir ces fonds plus tard sur le mainnet
```

**Comment by bitschmidty** [comment]
Pushed a small commit to removing trailing space and fix the build. @Copinmalin let me know when the newsletter is ready for review and out of draft mode.

**Comment by Copinmalin** [comment]
Thanks for your help @bitschmidty @murchandamus

**Comment by murchandamus** [comment]
I noticed that the title differed from the prior years, and that the `excerpt` variable had been translated. I renamed it to the original which fixed the intro paragraph.

**Comment by murchandamus** [comment]
Scrolled over it, now looks good visually. Did not check all the links this time, there are too many.

**Comment by murchandamus** [comment]
I checked a few links and scrolled over it a bit slower one more time. LGTM, we can fix any minor issues that might still be discovered. Thanks @Copinmalin and @Jluc-bitcoinfr

**Pull request merged**